run

```diff
@@ -14,7 +14,6 @@ fi
 python3 ./save_settings
 
 docker run -d \
-    --restart always \
```

Comment:
This doesn't work because you need to get the secrets from vault for the container to start successfully. So I've just removed this and will add instructions to `montagu-machine` reminding us that we need to manually start the webhook back up after a reboot.

Reply:
https://github.com/vimc/montagu-machine/compare/master...RESIDE-27